### DISCUSSION TITLE
Change  OIDCAuthRequestParams  value at request time dynamically

### DISCUSSION THREAD

**Discussion by gautamkct** [OpenIDC/mod_auth_openidc#839] [Q&A] [answered]
HI Team, 

We have mod_auth_openidc implemented on webserver for OIDC integration and its working fine however when we enable MFA along with CORS support it start failing due to x-frame-option defined at provider level which prevent CORS while MFA is enabled on source server so we tried to make conditional so that if request come from test.com  bypass MFA however this doesn't seems to be working. 

Any pointer will support.

In **auth_openidc.conf** file 

```
<If "req('Host') == 'test.com'"> 
	OIDCAuthRequestParams acr_values=<acr value goes here without MFA >"
</If>
<Else>
	OIDCAuthRequestParams acr_values=<acr value goes here with MFA>"
</Else>
```

**Comment by zandbelt** [comment, marked as answer]
`OIDCAuthRequestParams` is a server level directive which is evaluated only at startup, `OIDCPathAuthRequestParams` should work better because it is evaluated on every request

**Comment by gautamkct** [comment]
Thanks @zandbelt . So does it mean OIDCPathAuthRequestParams  can we used instead  OIDCAuthRequestParams .

Kindly suggest

**Reply by zandbelt** [reply]
yes

**Reply by gautamkct** [reply]
Thanks alot @zandbelt  for your support.
